Formula/t/three-body.rb

```diff
@@ -1,8 +1,8 @@
 class ThreeBody < Formula
   desc "三体编程语言 Three Body Language written in Rust"
   homepage "https://github.com/rustq/3body-lang"
-  url "https://github.com/rustq/3body-lang/releases/download/0.3.0/3body-lang.tar.gz"
-  sha256 "1dad812349ca941cb084815ad97f406a60eaed832e58a7adf4b8799f3bff4055"
+  url "https://github.com/rustq/3body-lang/releases/download/0.3.1-rc.0/3body-lang.tar.gz"
```

Comment:
we dont ship rc release

Reply:
got it

That is a fault PR, correct PR here [#152652](https://github.com/Homebrew/homebrew-core/pull/152652)